src/Tribe/Tickets.php

```diff
@@ -3204,5 +3263,7 @@ public function edit_global_capacity_level() {
 			_deprecated_function( __METHOD__, '4.6.2', 'wp_send_json_success()' );
 			wp_send_json_success( $data );
 		}
+
+		// @codingStandardsIgnoreEnd
```

Comment:
Just adding a note here, the ignore here is wrapped around the deprecated methods, it looks like this PR moves it so it only wraps the deprecated methods as it was in the wrong place before somehow.

Reply:
Right. I'm guessing someone ran PhpStorm reformat code, which I've noticed leaves `//` comments in place if not within a function but moves `/*` comments along with its connected function.